packages/editor/src/components/components.test.ts

```diff
@@ -11,15 +11,15 @@ test('componentByName', () => {
 test('componentByCategory', () => {
   const result = componentsByCategory('Elements');
   expect(result.General).toEqual(undefined);
-  expect(result.Input).toHaveLength(4);
+  expect(result.Input).toHaveLength(5);
   expect(result.Selection).toHaveLength(3);
   expect(result.Text).toHaveLength(1);
 });
 
 test('allComponentsByCategory', () => {
   const result = allComponentsByCategory();
```

Comment:
If you use the hidden category (other comment) you could add a simple check here that the hidden category will not appear in this object.